topcoffea/modules/event_selection.py

```diff
@@ -69,3 +69,19 @@ def get_Z_peak_mask(lep_collection,pt_window,flavor="os",zmass=91.2):
         raise Exception(f"Error: flavor requirement \"{flavor}\" is unknown.")
     sfosz_mask = ak.flatten(ak.any((zpeak_mask & sf_mask),axis=1,keepdims=True)) # Use flatten here because it is too nested (i.e. it looks like this [[T],[F],[T],...], and want this [T,F,T,...]))
     return sfosz_mask
+
+# Returns a mask for all events with any os lepton pair within low region of off-Z
+# Mask will be True if any combination of 2 leptons from within the given collection satisfies the requirement
+def get_off_Z_mask_low(lep_collection,pt_window,flavor="os"):
```

Comment:
If we want to generalize this a bit, we could remove `_low` from the name, and change `pt_window` to a string that takes `"low"` or `"high"`